### ISSUE TITLE
make it possible to add item attributes as key/value pairs, not just values

### ISSUE THREAD

**Issue by UncleTom** [reportportal/agent-python-pytest#185]
per search requirements on UI, hope we can split {"value":value} to {"key:key,"value":value}
see below local changes

```    
def _get_item_markers(self, item):
        # Try to extract names of @pytest.mark.* decorators used for test item
        # and exclude those which present in rp_ignore_attributes parameter
        def get_attribute(item, keyword):
            try:
                marker = item.get_closest_marker(keyword)
            except AttributeError:
                # pytest < 3.6
                marker = item.keywords.get(keyword)

            if marker and marker.args:
                return {"key": keyword, "value": marker.args[0]}
            else:
                return {"value": keyword}

        try:
            get_marker = getattr(item, "get_closest_marker")
        except AttributeError:
            get_marker = getattr(item, "get_marker")
        attributes = [get_attribute(item, k)
                      for k in item.keywords if get_marker(k) is not None
                      and k not in self.ignored_attributes]

        attributes.extend([{"value": tag} for tag in
                           item.session.config.getini('rp_tests_attributes')])
        return attributes
```

**Comment by iivanou** [comment]
@UncleTom If you have any suggestions, send them out as the PR. Do not forget to add unit tests.

**Comment by iivanou** [comment]
Fixed in #215.
